collect_app/src/main/java/org/odk/collect/android/dao/CursorLoaderFactory.java

```diff
@@ -0,0 +1,153 @@
+package org.odk.collect.android.dao;
+
+import androidx.loader.content.CursorLoader;
+
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.instances.Instance;
+import org.odk.collect.android.provider.InstanceProviderAPI;
+
+public class CursorLoaderFactory {
```

Comment:
Don't we need a test for this class?

Reply:
The code is being called from the same places but have now just moved into a new object. As far as I can recall there was no testing for the individual `CursorLoader` methods before, but they're all used as UI so should have a lot of implicit coverage from our Espresso tests 🤞. Is there any logic in particular here you felt might need a test?

Reply:
Since everything goes down to `getInstancesCursorLoader()` we could for example test that the method is called with proper parameters in all cases? but it's not a must I'm just thinking...

Reply:
Hmmmmm yeah especially for things like ordering it'd be nice to know we're calling the right thing. I do think because none of this has changed (the methods have just moved) and it's all things we want to move I wouldn't look at backfilling tests as a priority. I think it'd be hard to write tests for the specifics (like sort order) that wouldn't have to be changed if we remove cursor loaders. Does that make sense?

Reply:
Yeah